S12-enums/basic.t

```diff
@@ -252,4 +252,11 @@ is-deeply do { my enum Foos (a => <42>); a.Str }, 'a',
     'Pair elements in the list given to enum declaration work';
 }
 
+{ # RT #129142
+    enum FF <zero one two three>;
```

Comment:
It'd be better to have these as `my enum`, so they're restricted to the scope of that block and have no chance of affecting other tests that may get added to this file.